risedev.yml

```diff
@@ -732,6 +732,14 @@ profile:
         enable-tiered-cache: true
       - use: frontend
       - use: compactor
+  hummock-trace:
```

Comment:
Can you update the `src/storage/hummock_trace/README.md` so that we can try running it with this profile?